scripts/gen_build_no.sh

```diff
@@ -20,7 +20,7 @@ BUILD_NUMBER_FILE="${GIT_ROOT}/BUILD_NUMBER"
 if [[ -f "${BUILD_NUMBER_FILE}" ]]; then
     cat "${BUILD_NUMBER_FILE}"
 else
-    # Format: Year(2 digit) + Month + Day + Hour + Minutes
-    # Example: 1812011805
-    date '+%y%m%d%H%M' | tee "${BUILD_NUMBER_FILE}"
+    # Format: Year(2 digit) + Month + Day + Hour
+    # Example: 2018120118
```

Comment:
The comment is wrong now. Why this change, btw?

Reply:
I've removed minutes to reduce possibility of getting different build numbers for different platforms.

Reply:
Also, fixed comment.